test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/DifferentTestFrameworkSimpleTests.cs

```diff
@@ -124,21 +124,23 @@ public void RunTestsWithChutzpahAdapter(RunnerInfo runnerInfo)
         SetTestEnvironment(_testEnvironment, runnerInfo);
         Setup();
 
-        var sources = new List<string>
-        {
-            Path.Combine(_testEnvironment.TestAssetsPath, "test.js")
-        };
+        var jsSource = Path.Combine(_testEnvironment.TestAssetsPath, "test.js");
+
+        // Chuzpah adapter creates _Chutzpah temp files, to give data back from the runner.
+        // But when cleaning up it deletes all the _Chutzpah files, not just the one it owns,
+        // so when we run in parallel, the slower process will never find it's own file, because it was already deleted:
```

Comment:
Nit: Seems that you usually limit to around 100 chars your comments, this one goes a bit longer.

Reply:
That probably depends on how my VS is currently setup. When I have two windows next to each other I write shorter comments :D We should set some standard for this, if VS allows me to set horizontal line for max line size.